acceptance/tests/invalid_ssl_config.rb

```diff
@@ -4,10 +4,19 @@
 
 agent1 = agents[0]
 
+step 'Setup - Add base certs and config file'
+test_ssl_dir = agent1.tmpdir('test-ssl')
+scp_to(agent1, '../test-resources/ssl', test_ssl_dir)
+test_ssl_dir = File.join(test_ssl_dir, 'ssl')
+create_remote_file(agent1, pxp_agent_config_file(agent1), pxp_config_json_using_test_certs(master, agent1, 1, test_ssl_dir).to_s)
+if agent1['platform'] =~ /windows/
```

Comment:
config_helper.rb has a windows? method to test if a host is Windows; it could/should be re-used here